service.go

```diff
@@ -55,6 +55,7 @@ type API struct {
 		ReadHeaderTimeout time.Duration        `json:"read_header_timeout"`
 		WriteTimeout      time.Duration        `json:"write_timeout"`
 		IdleTimeout       time.Duration        `json:"idle_timeout"`
+		UIRedirect        string               `json:"ui_redirect"`
```

Comment:
Does this need to be config based, aka another configuration value and change, when the values seems hardwired?

Reply:
Can topaz pass a default so this is not required to be in the config file?